src/main/scala/firrtl/transforms/DedupAnnotations.scala

```diff
@@ -18,13 +18,16 @@ import firrtl.analyses.InstanceKeyGraph
 import scala.collection.mutable.ArrayBuffer
 
 object DedupAnnotationsTransform {
-
-  final class DifferingModuleAnnotationsException private (msg: String) extends PassException(msg)
-  object DifferingModuleAnnotationsException {
-    def apply(left: ReferenceTarget, right: ReferenceTarget): DifferingModuleAnnotationsException = {
-      val msg = s"${left.serialize} and ${right.serialize} have differing module binaries"
-      new DifferingModuleAnnotationsException(msg)
-    }
+  private def dedupAnno(annotation: Annotation): Option[(Any, Annotation, ReferenceTarget)] = annotation match {
+    case a @ MemoryRandomInitAnnotation(target) =>
+      Some(((target.pathlessTarget, Nil), a.copy(target = target.pathlessTarget), target))
```

Comment:
```suggestion
      Some((target.pathlessTarget, a.copy(target = target.pathlessTarget), target))
```
Small nit but no point returning a `(target.pathlessTarget, Nil)` when you could just return `target.pathlessTarget` (since that part of the tuple is an `Any` anyway.